.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,77 @@
+name: Generate Az ps module
+
+on:
+  workflow_dispatch:
+#   schedule:
+#     - cron: '*/5 * * * *'
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  publish_release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Finding version to publish
+        run: |
+          $version = ./Find-MinModuleVersionNotInManifest.ps1 -moduleName Az -versionsManifestPath ./versions-manifest.json
```

Comment:
We need to remove Find script as now the plan is to cache only the specific versions of the module that we plan to support. So instead take the specific version that we plan to cache as an input.